### ISSUE TITLE
ebreak instruction retires

### ISSUE THREAD

**Issue by tzwaenn** [chipsalliance/rocket-chip#2672]
**Type of issue**: bug report

**Other information**
 
The RISC-V privileged spec states:
As ECALL and EBREAK cause synchronous exceptions, they are not considered to retire, and should not
increment the minstret CSR.

**If the current behavior is a bug, please provide the steps to reproduce the problem:**
execute ebreak instruction

**What is the current behavior?**
minstret is incremented on ebreak

**What is the expected behavior?**
minstret should not be incremented
![retire](https://user-images.githubusercontent.com/45827187/96719731-15709580-13aa-11eb-999f-32b1bb28e20f.png)

**Please tell us about your environment:**
The issue was found with OneSpin 360 formal verification and the Processor Integrity App

**Comment by aswaterman** [comment]
I agree the EBREAK instruction never technically retires, but slight inaccuracy in some performance counters is a tolerable WONTFIX kind of erratum.
